content/projects/gsoc/index.adoc

```diff
@@ -119,7 +119,7 @@ During these time slots Jenkins GSoC org admins and mentors are available for an
 * Schedule: weekly 30 minutes meetings. Office hours will be held on Thursdays at 16:00 UTC.
   Use the link:/event-calendar[Jenkins event calendar] to view the meeting time in your own time zone.
 * link:https://docs.google.com/document/d/1UykfAHpPYtSx-r_PQIRikz2QUrX1SG-ySriz20rVmE0/edit?usp=sharing[Agenda]
-* Meetings are commonly recorded on-demand and posted link:https://www.youtube.com/playlist?list=PLN7ajX_VdyaO1f6bvkcSzW4PdWKkLktRG[here].
+* Meetings are commonly recorded on-demand and posted link:https://www.youtube.com/@jenkinscicd/playlists
```

Comment:
```suggestion
* Meetings are commonly recorded on-demand and posted link:https://www.youtube.com/@jenkinscicd/playlists[here]
```

Reply:
I think this is the proper syntax for the link to work.

Reply:
I also think we only need the playlist for GSoC 2024, but not the one for all of Jenkins.

Reply:
> I also think we only need the playlist for GSoC 2024, but not the one for all of Jenkins videos.

@krisstern You are right and I have implemented changes as requested.